src/js/MainController.js

```diff
@@ -95,6 +95,15 @@ goog.require('ga_topic_service');
         startWith3D = true;
       }
 
+      var onRenderError = function(scene, error) {
+        $scope.globals.is3dActive = undefined;
+        alert($translate.instant('3d_render_error'));
+        $window.console.error(error.stack);
```

Comment:
I'm in favor to keep this because it's a fatal error of cesium (we already use it when the inialisation failed). It's easier to see why the 3d doesn't work, and also because the native webgl msg is not so clear.